src/components/ClientQuote/ClientQuote.module.scss

```diff
@@ -0,0 +1,67 @@
+.main {
+  display: flex;
+  flex-direction: row;
+
+  @media (max-width: 768px) {
+    flex-direction: column-reverse;
+  }
+}
+
+.clientQuote {
+  padding: 30px 20px;
+  width: 30vw;
```

Comment:
`max-width` is usually better for containing text width. that way if it's ever shortened, it's not taking up extra space it doesnt need